docs/reference/changelog.rst

```diff
@@ -2,6 +2,17 @@
 Changelog
 *********
 
+.. _release-3.2.1:
+
+3.2.1 (2025-MM-DD)
```

Comment:
If you'll change this to today I'll release it today.

Reply:
Dated!